manuals/1.16/index.md

```diff
@@ -69,16 +69,16 @@ previous_version: "1.15"
 {%include manuals/{{ page.version }}/4_general_foreman.md %}
 ## 4.1 Web Interface
 
-### 4.1.1 LDAP Authentication
-{%include manuals/{{ page.version }}/4.1.1_authentication.md %}
-### 4.1.2 Roles and Permissions
-{%include manuals/{{ page.version }}/4.1.2_roles_and_permissions.md %}
-### 4.1.3 Trends
-{%include manuals/{{ page.version }}/4.1.3_trends.md %}
-### 4.1.4 Auditing
-{%include manuals/{{ page.version }}/4.1.4_auditing.md %}
-### 4.1.5 Searching
-{%include manuals/{{ page.version }}/4.1.5_searching.md %}
+### 4.1.1 Searching
```

Comment:
This would require also updating references to it from the foreman itself (e.g. https://github.com/theforeman/foreman/blob/698e916ce208b5040b83a908a058c83c94d158ee/app/views/bookmarks/welcome.html.erb). The problem is we don`t have nightly version for docs, that would make it easier to do this changes in sync with the code. @dlobatog @GregSutcliffe thoughts?

Reply:
I don't get why that move is required?

Reply:
@mmoll , the section of "Roles and Permissions" are already explain about filters and bookmarks, but there is no mention about it prior to it, so you encounter syntax operands like `^`, and you do not know what it means.

But if you first read about it, then even if you do not remember it meaning, you still remember that it is written in some place.

For example I didn't know the syntax, so I'ved asked about it one of my team members, and he addressed me to the right wiki to look at it.

Reply:
@iNecas, I'm willing to do the "dirty work" for this :)
That is, to look for all sections that needed to change, and change them.

Reply:
The problem right now is we dont have a nightly version (note we are editng 1.16 version while we already branched 1.17) of docs in github right now, so the question is how to make this changes and keep them in sync.

Anyway, for the issue itself, just putting a link to the search section from other parts might work as well, plus it would help people, that don't go through the docs sequentially

Reply:
@iNecas what is the process for documenting 1.17?
Does someone copy v1.16 and change stuff based on 1.17 changes?

Reply:
That is my question for @dLobatog or @GregSutcliffe

Reply:
There's no automatic process for it, usually it's added to the PR, or a separate PR is added to the checklist for the next release of the docs.

Reply:
As to the subject of nightly docs, IIRC the N+1 docs are branched at the same time as the code (that is, 1.18 docs should be available for commits once 1.18 itself is branched). We could create a non-deployed branch for nightly stuff easily enough, but deploying that may be messy. Feel free to start a dev-discussion on this and I'll have a think :)